src/mito2/src/sst/file.rs

```diff
@@ -312,6 +312,14 @@ impl FileMeta {
         !self.available_indexes.is_empty()
     }
 
+    pub fn index_version(&self) -> Option<IndexVersion> {
+        if self.exists_index() {
+            Some(self.index_version)
+        } else {
+            None
+        }
+    }
```

Comment:
question: Should we return 0 as index version?

Reply:
> question: Should we return 0 as index version?

no? This differ between having a index/not having one?